.env

```diff
@@ -0,0 +1,9 @@
+COAP_CLI_HOST=localhost
+COAP_CLI_PORT=5683
+COAP_CLI_CONTENT_FORMAT=50
+COAP_CLI_AUTH=
+COAP_CLI_OBSERVE=
+COAP_CLI_CERT_FILE=
```

Comment:
Since this is a cli application probably configuring it in Toml is better also the location should be ideal. You can use https://github.com/muesli/go-app-paths